cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -701,6 +701,9 @@ func Test_objectMover_backupTargetObject(t *testing.T) {
 				}
 
 				fmt.Printf("Actual file content %v\n", string(fileContents))
+				if string(fileContents) != expectedFileContents {
+					fmt.Printf("test")
+				}
```

Comment:
Left over?

Reply:
Thx! yes :/
Fixed

Reply:
Used it as a way to be able to add a break point in the failure case to be able to easily diff with the debugger